src/tags/control/Rectangle.js

```diff
@@ -44,12 +45,18 @@ const TagAttrs = types.model({
   canrotate: types.optional(types.boolean, true),
 });
 
+const allowedToolNames = ['Rect'];
```

Comment:
Rect.name and Rect3Point.name evaluate to different values. so that wont work here unfortunately and changing these values will cause the tools to disappear from the toolbar - so unfortunately that wont work.

I do like the suggestion so I'd recommend we address this when we refactor the file